### PR TITLE
Update seeders to add revisions and activity

### PR DESCRIPTION
**Pull request by driftingly**
This PR updates the seeding methods to add revisions and activity.

Screenshot 2024-05-09 at 10 52 36 AM

### CODE REVIEW

File: app/Repositories/ArtworkRepository.php [hunk 1]
```diff
@@ -8,11 +8,15 @@
 use A17\Twill\Repositories\ModuleRepository;
 use App\Libraries\Api\Builders\ApiQueryBuilder;
 use App\Models\Artwork;
+use App\Repositories\Traits\HandleDynamicArtworkFields;
 use Illuminate\Support\Str;
 
 class ArtworkRepository extends ModuleRepository
 {
-    use HandleMedias, HandleRevisions, HandleTranslations;
+    use HandleDynamicArtworkFields;
```
Comment (driftingly) [inline app/Repositories/ArtworkRepository.php:16]: Saving artwork was causing a `Unknown column 'thumbnail' in 'field list'` errors.
These trait methods are called by the base `ModuleRepository` to clean up the fields before saving or updating.

File: database/seeders/ThemeSeeder.php [hunk 2]
```diff
@@ -36,26 +38,34 @@ public function __construct(
...
+                    ],
+                ])->merge($rawTheme['translations'])->map(
+                    fn ($translation, $locale) => [
+                        'title' => [$locale => $translation['title']],
+                        'intro' => [$locale => $translation['intro']],
+                        'journey_guide' => [$locale => $translation['journey_guide']],
+                    ]
+                )->reduce(function (array $carry, array $translation) {
+                    return array_merge_recursive($carry, $translation);
+                }, []);
```
Comment (driftingly) [inline database/seeders/ThemeSeeder.php:61]: This changes the shape of data to group by field:

```php
[
    'title' => [
        'en' => 'Example Title',
        'es' => '...',
        'zh' => '...',
    ],
    // ...
]
```

File: app/Providers/AppServiceProvider.php [hunk 3]
```diff
@@ -34,6 +35,7 @@ public function boot(): void
             'theme' => Theme::class,
             'theme_prompt' => ThemePrompt::class,
             'artwork' => Artwork::class,
+            'prompt_artwork' => ThemePromptArtwork::class,
```
Comment (driftingly) [inline app/Providers/AppServiceProvider.php:38]: Added to support activity logs.

**Review by nikhiltri — APPROVED** [review]
Looks good! ⭐

File: database/seeders/ThemeSeeder.php [hunk 4]
```diff
@@ -36,26 +38,34 @@ public function __construct(
 
     public function run(): void
     {
+        // Log in as admin to attribute activity to them
+        Auth::guard('twill_users')->login(User::find(1));
+
```
Comment (zachgarwood) [inline database/seeders/ThemeSeeder.php:43]: 👍

**Review by zachgarwood — APPROVED** [review]

**Pull request merged**